presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1176,6 +1177,12 @@ protected RelationType visitQuerySpecification(QuerySpecification node, Analysis
     @Override
     protected RelationType visitUnion(Union node, AnalysisContext context)
```

Comment:
Ya removed it. Also updated DefaultTraversalVisitor to have `visitSetOperation` since this one extends `DefaultTraversalVisitor`